### PR TITLE
Fix README.rst, create website page, document using and developping 

### PR DESCRIPTION
**Pull request by tsirif**
Related to issue #39

Not yet completed. Pull requesting just to get the touch of things

### CODE REVIEW

**Comment by bouthilx** [comment]
@tsirif Suppose we go with the work division I propose in issue #39, what would be done for part 1 in the current PR?

### 1. Base doc

- [ ]  API and code reference
- [ ]  Installation & deployment process

**Comment by bouthilx** [comment]
@tsirif Ping :)

**Comment by bouthilx** [comment]
@tsirif Is anything still missing beside the error to fix? Can you check one of the boxes [here](https://github.com/mila-udem/orion/pull/59#issuecomment-379345799)?

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-sphinx >= 1.6.5, < 2
+sphinx >= 1.6.5, < 1.7.3
```
Comment (bouthilx) [inline docs/requirements.txt]: It helps to also write **why** it breaks in the commit message. :wink:

Comment (tsirif) [reply docs/requirements.txt]: Well, I don't know the reason :stuck_out_tongue_closed_eyes:

Comment (bouthilx) [reply docs/requirements.txt]: In this case you can write "I don't know why". :stuck_out_tongue: At least we know we don't know. :sweat_smile:

**Comment by bouthilx** [comment]
@tsirif Needs rebase

**Comment by dendisuhubdy** [comment]
@tsirif @bouthilx guys should we merge this one too? I was asked by some friends in SV that the description of the pages looks kinda ugly, I see that this fixes the Readme.rst and the docs.

**Comment by bouthilx** [comment]
Waiting for fix to this comment https://github.com/mila-udem/orion/pull/59#discussion_r184114563

File: README.rst [hunk 2]
```diff
@@ -1,57 +1,89 @@
...
+- Algorithms algorithms algorithms:
+  Skopt_'s bayesian optimizers are at hand without writing.
+  Random search is the default.
+  **only** a single line of code.
+- More algorithms:
+  Implementing and distributing algorithms is as easy as possible!
+  Check `developer's guide-101`_. Expect algorithm plugins to pop out quickly!
+- Came up with an idea?
+  Your intuition is still at play:
+  Help your optima hunter now by a command line interface.
```
[the remark this answers is not in the document]
Comment (tsirif) [reply README.rst]: Let's update this in a later PR, when #70 is merged

**Comment by tsirif** [comment]
@dendisuhubdy what's SV?

**Comment by bouthilx** [comment]
Silicon Valley? :thinking:

**Comment by bouthilx** [comment]
Rebased in #78

**Pull request closed without merging**